MetaCell/salk-interactive-atlas
# Feature/190 grid

Closes #190 2, 3, 6, 7 and 8

Implemented solution: 
- Generates grid dimensions with a custom rule, based on atlas shape and using 0.25 as tick step. (For the salk atlas this results in a 2.5mm x 2mm grid, as requested)
- Tick step fixed; Previously version was using a step on the grid scale but the ticks are happening at the figure scale; img_to_res_ratio variable contains the conversion rate between the two.
- Updates grid metadata (title, axis label)
- Updates Z depth point of reference to C8-Caudal
- Pads/Shifts images (annotation, lamina, canal) so that the center of the grid matches with the center of the canal)
- Stores grid dimensions in metadata.json

- Updates tests locations

- Reads canvas dimension from atlas static data


How to test this PR: 
- Run the application
- Make sure the grid looks readable and accurate in terms of dimensions considering the center as the center of the canal

### Sanity checks:

### Breaking changes (select one):

### Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):

![image](https://user-images.githubusercontent.com/19196034/186955973-0e50601e-c709-489e-b738-adc72cc3d7bd.png)
